Cargo.toml

```diff
@@ -231,6 +235,9 @@ wasm-timings = ["wasm"]
 # transforms and sinks should depend on this feature.
 kubernetes = ["k8s-openapi", "evmap"]
 
+# Enables the API
+api = ["warp", "async-graphql", "async-graphql-warp"]
```

Comment:
Perhaps we can come up with a more descriptive name for this feature? `internal-events-api` or something?

Reply:
There's probably a better name for this than just 'api', although my starting intention is that this would become the de facto gateway into general UI <-> Vector observability and grow into more than just events.

One current use-case we're planning for, for example, is retrieving/validating _Cargo.toml_.

Reply:
To conclude this point: In the absence of a better name, I'll keep this as `api` for now. I do think it'll grow to be more than simply metrics, so trying to keep the nomenclature in sync with its function might be a bit tough; this generalisation may be preferred.